test-network-k8s/docs/CA.md

```diff
@@ -0,0 +1,281 @@
+    1. [Launch the ECert CA Servers](https://hyperledger-fabric-ca.readthedocs.io/en/latest/deployguide/cadeploy.html#start-the-ca-server)
+    1. [Enroll the ECert CA Bootstrap / Admin User](https://hyperledger-fabric-ca.readthedocs.io/en/latest/deployguide/cadeploy.html#enroll-the-ca-admin)
+
+
+## [Deploy the TLS CAs](https://hyperledger-fabric-ca.readthedocs.io/en/latest/deployguide/cadeploy.html#deploy-the-tls-ca)
+
+### [Configure the TLS CA Servers](https://hyperledger-fabric-ca.readthedocs.io/en/latest/deployguide/cadeploy.html#modify-the-tls-ca-server-configuration) 
+
+While the CA guide suggests running the `fabric-ca-server` binary to generate a default configuration file, for the 
+test network we've skipped this step and have added a [config/fabric-tls-ca-server-config.yaml](../config/org0/fabric-tls-ca-server-config.yaml) 
+to the top level of this project.  
+
+Changes have been made to reflect: 
+
+- `port: 443` binds all traffic to the default HTTPS port
+- `tls.enabled: true` enables TLS for registration and enrollment requests
+- `ca.name: <service-name>` matches the Kubernetes `Service` host alias 
+- `csr.hosts:` includes host aliases for accessing the CA with Kube DNS 
+
+
+Prior to launching the CA, for each org we create a configmap including the TLS CA server yaml: 
+
+```shell
+kubectl -n test-network create configmap org0-config --from-file=config/org0
+kubectl -n test-network create configmap org1-config --from-file=config/org1
+kubectl -n test-network create configmap org2-config --from-file=config/org2
+```
+
+
+### [Launch the TLS CA Servers](https://hyperledger-fabric-ca.readthedocs.io/en/latest/deployguide/cadeploy.html#start-the-tls-ca-server)
+
+```shell
+✅ - Launching TLS CAs ...
+```
+
+For each org we create a Kube Deployment and Service, ensuring that the org config 
+map and persistent volume maps to the correct location on disk.   
+
+```shell
+kubectl -n test-network apply -f kube/org0/org0-tls-ca.yaml
+kubectl -n test-network apply -f kube/org1/org1-tls-ca.yaml
+kubectl -n test-network apply -f kube/org2/org2-tls-ca.yaml
+```
+
+As a side-effect of bootstrapping the TLS CA, each storage volume will include a self-signed certificate 
+pair to serve as the **Root TLS Certificate**.  Pay special attention to this path, as it will be used extensively 
+to verify the TLS host name of all services within the organization: 
+```shell 
+${FABRIC_CA_CLIENT_HOME}/tls-root-cert/tls-ca-cert.pem 
+```
+
+
+### [Enroll the TLS CA Bootstrap Admin Users](https://hyperledger-fabric-ca.readthedocs.io/en/latest/deployguide/cadeploy.html#enroll-bootstrap-user-with-tls-ca)
+```shell
+✅ - Enrolling bootstrap TLS CA users ...
+```
+
+After the TLS server is running, we need to enroll the bootstrap admin user with the CA.  This admin user will 
+then be employed to fulfill a Certificate Signing request for the ECert CA servers, allowing for full host 
```

Comment:
eCert - have a memory that was a fabric 0.6 only term?

Reply:
Unclear.   Written by a Fabric `n00b` - trying to be as consistent as possible with the terms in the HLF docs / guides.